aiohttp/web_exceptions.py

```diff
@@ -420,18 +420,21 @@ class HTTPUnavailableForLegalReasons(HTTPClientError):
 
     def __init__(
         self,
-        link: StrOrURL,
+        link: Optional[StrOrURL] = None,
         *,
         headers: Optional[LooseHeaders] = None,
         reason: Optional[str] = None,
         text: Optional[str] = None,
         content_type: Optional[str] = None,
     ) -> None:
+        if link is not None and not link:
+            raise ValueError("link argument cannot be empty.")
```

Comment:
Do you think this is worth checking? We could just treat an empty str the same as None.

Reply:
I implemented it that way to be consistent with other exceptions accepting URL arguments.
I can make it just `if link:` if you prefer. Let me know!

Reply:
Which other ones are you referring to? The only similar one I see is location in HTTPMove, where the url is mandatory.

Reply:
Yes I meant the HTTPMove exception indeed, in particular this check:
```python
        if not location:
            raise ValueError("HTTP redirects need a location to redirect to.")
```
That only accepts `location` if not None _and_ not empty. Which is what I tried to mirror with `link`.

Reply:
I will rework the check 👍

Reply:
Implemented the changes in the recent version of this PR.